collector/eventhub/src/main/java/zipkin/collector/eventhub/EventHubCollector.java

```diff
@@ -33,15 +28,18 @@
 public class EventHubCollector implements CollectorComponent {
 
   private Builder builder;
```

Comment:
the builder is mutable, so shouldn't be saved. If you look at other implementations, you'll notice the builder fields are copied into the resulting collector. If you prefer me to help adjust this code, no problem!